### PR TITLE
add rust binding and demos

### PR DESCRIPTION
**Pull request by ksyeo1010**
- create rust binding and demo

NOTE: will build package and release after approval so demos workflow work.

### CODE REVIEW

File: binding/rust/src/leopard.rs [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+            return Err(LeopardError::new(
+                LeopardErrorStatus::ArgumentError,
+                format!(
+                    "Could not find the audio file at `{}`",
+                    audio_path.as_ref().display()
+                ),
+            ));
+        }
+
+        let transcript: String;
```
Comment (ErisMik) [inline binding/rust/src/leopard.rs:359]:
```suggestion
        let transcript = String::new();
```
Let's not leave any uninitialized variables

**Comment by ErisMik** [comment]
Before merging can you run `cargo fmt` in the binding and demos?

File: binding/rust/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  - macOS (x86_64, arm64)
+  - Windows (x86_64)
+  - Android
+  - iOS
+  - Raspberry Pi (4, 3)
+  - NVIDIA Jetson Nano
+
+## Requirements
+
+- .NET Core 3.1
```
Comment (mrrostam) [inline binding/rust/README.md:23]: I believe this is incorrect

File: demo/rust/filedemo/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "pv_leopard_filedemo"
+version = "0.1.0"
```
Comment (mrrostam) [inline demo/rust/filedemo/Cargo.toml:3]:
```suggestion
version = "1.0.0"
```

**Review by mrrostam — APPROVED** [review]

**Pull request merged**